### DISCUSSION TITLE
Coolify v4.0.0 stabilization for production use (Feature freeze).

### DISCUSSION THREAD

**Discussion by nuxwin** [coollabsio/coolify#3536] [General]
Good morning,

We are using **Coolify** in our **preview** infrastructure and we now want use it in our **production** environment but ... The **v4.0.0** is still in **beta** state, and as we can see, there is no freeze yet as new features are still added. Lately, we discovered bugs which were not in previous versions and this is a bit annoying for us.

Could it be possible to freeze the **v4.0.0** now, stabilizing it and releasing a production ready version ? That could be great. New features could go for **v5.0.0** or **v4.1.0**, depending on how you're seeing the things ;)

Thank you for your understanding.

**Comment by peaklabs-dev** [comment]
That is a good idea. We were thinking about that as well, and we are proud to announce that we are doing a feature freeze from now on until we get to v4.0.0 stable. New features will be added to v4.1.0 and so on. Thanks for the suggestion.
